tests/test_diagnostics_report.py

```diff
@@ -0,0 +1,25 @@
+import os
+from dotenv import load_dotenv
+from supportbot.utils.diagnostics_report import get_report, lbe_traceroute_report
+
+load_dotenv()
+
+# commented out so test doesn't fail off Mesh
```

Comment:
Maybe we could change this so it's a conditional check? Shouldn't be that hard to just wrap things in a try-execept?

Reply:
Could also use an environment variable that gets set in the build system but not on the dev's laptop

Reply:
I ended up adding the `@pytest.mark.skipif`